### ISSUE TITLE
Captures go to current directory with a file name of "0"

### ISSUE THREAD

**Issue by jmzagorski** [dhruvasagar/vim-dotoo#75]
Hello i just started with your plugin and am having odd behavior when I perform a capture on `Phone Call`, `Meeting`, `Habit` or `Note`. When i save, write to a file called "0" in the current working directory. I believe `Todo` works as expected since it goes into my refile filebelow. Even when i remove the custom config i get the same behavior.

Windows 10 using gvim 8.2 Patch 1-981

```
let g:dotoo#agenda#files = ["~/org/*.org"]
let g:dotoo#capture#refile = expand("~/org/refile.org")

```
```
" this is set in my ftdetect folder
autocmd BufNewFile,BufRead *.org setf dotoo
```

Thanks.

**Comment by dhruvasagar** [comment]
@jmzagorski Thanks, there are some edge cases that produce these at times. Can you share your vim version ? I will investigate.

**Comment by jmzagorski** [comment]
Thanks for the response:

v8.2 patch 1-981, using gvim on windows. I normally recompile to the latest version every week or so.

**Comment by etrmrh** [comment]
I want to chime in and say I am experiencing the same issue. When I try to capture, the entry is saved to some random file not specified by my `g:dotoo#capture#refile`. Usually that's either "0" or "refile" in my current directory. 

Additionally, my refile menu will show "No refile!" even after I capture something or even if I move the headline manually to my specified `g:dotoo#capture#refile` file.

I'm using NVIM v0.4.4. Relevant settings in my config:

```
augroup dotooft
	autocmd!
	autocmd BufRead,BufNewFile *.org set filetype=dotoo
augroup END

let g:dotoo#agenda#files=['~/Org/*.org']
let g:dotoo#capture#refile=expand('~/Org/inbox.org')
let g:dotoo#capture#templates = {
			\ 't': {
			\   'description': 'Todo',
			\   'lines': [
			\	 '** TODO %?',
			\	 'DEADLINE: <%(strftime(g:dotoo#time#datetime_format))>'
			\   ],
			\  'target': 'refile:Tasks'
			\ },
			\ 'n': {
			\   'description': 'Note',
			\   'lines': ['* %? :NOTE:'],
			\ }
			\}
```

I think this is also related to issue #68

**Comment by stuart-warren** [comment]
I'm also getting this issue

**Comment by dhruvasagar** [comment]
I have pushed a fix for #68, that should fix this as well. Let me know if it's still an issue with the latest changes.
